jit-service/src/main/java/org/dcsa/jit/service/TimestampDefinitionService.java

```diff
@@ -125,6 +101,27 @@ public PortCallPhaseTypeCode findPhaseTypeCodeFromOperationsEventForJit1_0(
     return jit1_0.stream().findFirst().get().getPortCallPhaseTypeCode();
   }
 
+  public PortCallPhaseTypeCode findOmittedPhaseTypeCodeFromOperationsEventForJit1_1(
+      OperationsEvent operationsEvent) {
+
+    // JIT 1.1 use-cases with omitted port call phase type code
+    List<String> phaseTypeCodeOmitted = IntStream.range(5, 13).boxed().map(i -> "UC" + i).toList();
```

Comment:
These timestamps should be corrected in the IM til be from `jit1_0` instead.   I will do that  in my next IM update